index.bs

```diff
@@ -3967,6 +3976,38 @@ Note: The {{UserVerificationRequirement}} enumeration is deliberately not refere
         interest of minimizing disruption to the user interaction flow).
 </div>
 
+### Client Capability Enumeration (enum <dfn enum>ClientCapability</dfn>) ### {#enum-clientCapability}
+
+<xmp class="idl">
+    enum ClientCapability {
+        "user-verifying-platform-authenticator",
+        "hybrid-transport",
+        "passkey-platform-authenticator",
+        "conditional-mediation",
+        "conditional-create"
+    };
+</xmp>
+
+This enumeration defines a limited set of client capabilities which a [=[WRP]=] may evaluate to offer certain workflows and experiences to users.
+
+Note: The {{ClientCapability}} enumeration is deliberately not referenced, see [[#sct-domstring-backwards-compatibility]].
+
+<div dfn-type="enum-value" dfn-for="ClientCapability">
+    :   <dfn>user-verifying-platform-authenticator</dfn>
+    ::  The [=WebAuthn Client=] supports usage of a [=user-verifying platform authenticator=].
+
+    :   <dfn>hybrid-transport</dfn>
+    ::  The [=WebAuthn Client=] supports usage of the {{AuthenticatorTransport/hybrid}} transport.
+
+    :   <dfn>passkey-platform-authenticator</dfn>
+    ::  The [=WebAuthn Client=] supports usage of a [=passkey platform authenticator=], locally and/or via {{AuthenticatorTransport/hybrid}} transport.
```

Comment:
This description doesn't really align with our definition of [platform authenticator](https://w3c.github.io/webauthn/#platform-attachment). In [§6.2.1. Authenticator Attachment Modality](https://w3c.github.io/webauthn/#sctn-authenticator-attachment-modality), following the definition of _platform authenticator_, we state (emphasis added):

>Some [platform authenticators](https://w3c.github.io/webauthn/#platform-authenticators) could possibly also act as [roaming authenticators](https://w3c.github.io/webauthn/#roaming-authenticators) depending on context. For example, a [platform authenticator](https://w3c.github.io/webauthn/#platform-authenticators) integrated into a mobile device could make itself available as a [roaming authenticator](https://w3c.github.io/webauthn/#roaming-authenticators) via Bluetooth. In this case [clients](https://w3c.github.io/webauthn/#client) running on the mobile device would recognise the authenticator as a [platform authenticator](https://w3c.github.io/webauthn/#platform-authenticators), while **[clients](https://w3c.github.io/webauthn/#client) running on a different [client device](https://w3c.github.io/webauthn/#client-device) and communicating with the same authenticator via Bluetooth would recognize it as a [roaming authenticator](https://w3c.github.io/webauthn/#roaming-authenticators)**.

So the sentence "using a platform authenticator via hybrid transport" doesn't make sense with these definitions, because an authenticator reached via hybrid transport is by definition not a platform authenticator from the perspective of the client (even if it is a platform authenticator from the perspective of a different client device). Likewise, the definition of [passkey platform authenticator](https://w3c.github.io/webauthn/#passkey-platform-authenticator) is similarly inconsistent with its own name.

I don't think we currently have a term to describe this distinction between a roaming authenticator embedded in a different client device and a distinct roaming authenticator not embedded in any client device. I think we need a better name for this than "passkey platform authenticator", but that issue may be out of scope for this PR.

Reply:
I'm not sure how to implement this enum value unless it's just `hybrid-transport` OR `user-verifying-platform-authenticator`. Is it? If so, it that worthwhile?

Reply:
It's an either/or, from the PR that added `isPPAA()`:

![image](https://github.com/w3c/webauthn/assets/5166470/5049ca43-a456-4e30-a617-3a61fd65d3c4)

As for the value of this, it's intended to be convenient for an RP to use this one value than to have to know to write the either/or check themself. They could achieve the same thing using the uvpa and hybrid capabilities.

Reply:
For posterity I pulled this from https://github.com/w3c/webauthn/pull/1901